### PR TITLE
Updated readme to include instructions for setting environment path i…

### PR DESCRIPTION
**Pull request by sillyinfidel**
Just adds instructions for setting environment variables in win10

### CODE REVIEW

**Review by geekyed — COMMENTED** [review]
`";yourpath"` does this means it will add to the beginning of the path? If so would it be better to add to the end?

**Comment by sillyinfidel** [comment]
> 
> 
> `";yourpath"` does this means it will add to the beginning of the path? If so would it be better to add to the end?

The argument appends ;yourpath to the existing system paths, meaning it's added at the end of the statement. The semicolon serves as the delimiter between the new python path and existing paths in :path

**Review by geekyed — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,6 +33,14 @@ pipenv shell
 pipenv install
 ```
 
+```
+NOTE:
+If you are using Windows 10 and you have a fresh Python install, you need to set your environment paths.
+To do this, open Powershell and type $env:PATH += ";yourpath" 
+This command appends a semicolon + your path to the system path file.
+You can verify this worked by typing $env:PATH
+```
```
Comment (WillsB3) [inline README.md:42]: Please correct me if I'm wrong, but I'm not sure if this persists the PATH change across reboots. Have you tested this?

**Review by Hari-Nagarajan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Hari-Nagarajan) [reply README.md:42]: I was under the impression that on windows you need to edit the environment variables in this UI for it to persist:
image

Comment (WillsB3) [reply README.md:42]: So was I…

**Pull request closed without merging**